src/sentry/static/sentry/app/views/issueList/congratsRobots.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import background from 'app/../images/congrats-robots/background.png';
+import main from 'app/../images/congrats-robots/main.json';
```

Comment:
Can we code split this whole component actually, probably at a higher level (where it's used) so that this import doesn't end up in our main bundle.